src/components/molecules/TableView.tsx

```diff
@@ -60,6 +61,13 @@ const TableView: FC<IProps> = ({ data }) => {
     items.map((item) => item.injured_count),
   );
 
+  const accidentsByAscDate = [...items];
+  accidentsByAscDate.sort((a, b) => {
+    let aa = a.date.split('/').reverse().join();
```

Comment:
1. vars should have meaningful names (a,b are somewhat acceptable for sort function. aa and bb are not)
2. The proper way of sorting a date is converting the item to `Date` and comparing them